skimage/segmentation/_quickshift.pyx

```diff
@@ -112,7 +113,7 @@ def quickshift(image, ratio=1.0, kernel_size=5, max_dist=10,
                             dist += (current_pixel_p[channel] -
                                      image_c[r_, c_, channel])**2
                         dist += (r - r_)**2 + (c - c_)**2
-                        densities[r, c] += exp(-dist / (2 * kernel_size**2))
+                        densities[r, c] += exp(-dist / (2 * kernel_size_c**2))
```

Comment:
How about `cdef float kernel_size_sq = kernel_size**2` instead (it was a float before, and this saves taking the square each time).